feedback-module/src/components/Module.js

```diff
@@ -101,10 +96,20 @@ function Module({ pageTitle, endpoint }) {
         return { question: question, answer: answer };
       });
       setFeedbackForAPI(feedback);
-      sendFeedback();
+      sendRequest("feedback", {
+        id: endpoint,
+        feedback: feedbackForAPI,
+      });
       console.log(feedbackForAPI);
     } else if (formID === "research") {
-      setUserInfo(inputQuestions);
+      let userObj = userInfo;
+      inputQuestions.forEach(({ question, answer }) => {
+        userObj[question] = answer;
+      });
+      userObj.source = window.location.href;
```

Comment:
Did Justin say that he also wanted the window location to be included in the feedback data as well?

Reply:
I believe so— not sure if he meant the full URL or just the path though

Reply:
So we should probably also add it to the feedback object as a source field right? We can ask Justin if he prefers the full URL or just the path.

Reply:
Oh wait sorry I misread you comment— my bad! I don't think he said anything about adding it for the feedback data, just for user research if I remember correctly.

Reply:
Oh I thought for both but totally could be wrong. Should we check with him?

Reply:
Yeah it's def possible I remembered wrong lol I'll check with him